packages/block-library/src/image/editor.scss

```diff
@@ -27,13 +26,25 @@
 		&::before {
 			opacity: 0;
 		}
+
+		.block-bindings-media-placeholder-message {
+			opacity: 1;
+		}
+	}
+
+	.block-editor-media-placeholder.is-small {
+		min-height: 60px;
+	}
+
+	.block-editor-media-placeholder:not(.is-small) {
+		.components-placeholder__fieldset,
+		.components-placeholder__label {
+			display: flex;
+		}
```

Comment:
`.components-*` class names are meant to be a private implementation detail and should not be used outside of the components package.

Overrides like the one highlighter make this usage `Placeholder` component prone to future breakage, should the component update internally (either its classnames, or how its internal layout is organized).

Having said that, I can see that there are more examples of using private classnames and in general applying style overrides in this file, which is a strong indicator that we should completely rethink how the media placeholder is implemented. So, in that sense, my feedback shouldn't considered blocking for this specific PR, but rather a warning about how badly styles are architected for this component and the risks.

It feels like it's trying to bend the `Placeholder` component beyond its limits. Therefore:

- we should determine if there are any features that we can add to the `Placeholder` component so that consumers don't have to resort to style overrides and other hacky practices;
- otherwise, we should re-implement the component in a way that builds _on top_ of `Placeholder` . For example, instead of setting `display:none` to `.components-placeholder__illustration`, why don't we pass the `withIllustration={false}` prop?
- finally, if the designs / feature are diverging too much, we should ask ourselves whether this warrants a new, separate component (in the block library package)